io.ossia.score.yml

```diff
@@ -0,0 +1,333 @@
+
+  # Plugins
+  - --env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2:/app/lib/lv2
+
+  # Config persistence
+  - --persist=ossia
+
+add-extensions:
+  # Linux audio plugins base extension
+  org.freedesktop.LinuxAudio.Plugins:
+     directory: extensions/Plugins
+     version: '23.08'
+     add-ld-path: lib
+     merge-dirs: vst;vst3;lv2
+     subdirectories: true
+     no-autodownload: true
+
+  # Full codec support for FFMPEG
+  org.freedesktop.Platform.openh264:
+    directory: lib/openh264,
+    add-ld-path: .,
+    version: "2.4.1"
+
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg,
+    add-ld-path: .
+    version: "23.08"
+
+cleanup:
+  - /include
+  - /share/gir-1.0
+  - /share/man
+  - '*.a'
+  - '*.la'
+  - /lib/cmake
+  - /lib/girepository-1.0
+  - /lib/pkgconfig
+  - /lib/*.la
+  - /lib/*.a
+  - /share/pkgconfig
+  - /share/aclocal
+  - /share/gtk-doc
+  - /share/doc
+  - /share/info
+  - /share/man
+  - /man
+modules:
+  # LV2 audio plugin support
+  - shared-modules/linux-audio/lv2.json
+  - shared-modules/linux-audio/lilv.json
+
+  # Audio I/O
+  - name: portaudio
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+      - -DBUILD_SHARED_LIBS=OFF
+      - -DPA_ALSA_DYNAMIC=ON
+      - -DPA_USE_JACK=OFF
+    sources:
+      - type: archive
+        url: https://github.com/PortAudio/portaudio/archive/refs/tags/v19.7.0.tar.gz
+        sha256: 5af29ba58bbdbb7bbcefaaecc77ec8fc413f0db6f4c4e286c40c3e1b83174fa0
+
+  # Pitch shifting and resampling
+  - name: libsamplerate
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+      - -DBUILD_SHARED_LIBS=OFF
+      - -DBUILD_TESTING=0
+    sources:
+      - type: archive
+        url: https://github.com/libsndfile/libsamplerate/archive/4858fb016550d677de2356486bcceda5aed85a72.zip
+        sha256: 11af7a8243dfb81e211ed140139d3439b39ffc7527b92d5c5c4a5e7338f595ee
+
+  # Used for some audio file formats not supported by FFMPEG, reading ACID tags, etc.
+  - name: libsndfile
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+      - -DBUILD_SHARED_LIBS=OFF
+      - -DBUILD_PROGRAMS=0
+      - -DBUILD_EXAMPLES=0
+      - -DENABLE_CPACK=0
+      - -DENABLE_PACKAGE_CONFIG=0
+      - -DINSTALL_PKGCONFIG_MODULE=0
+      - -DINSTALL_MANPAGES=0
+      - -DENABLE_BOW_DOCS=0
+      - -DENABLE_EXPERIMENTAL=0
+      - -DENABLE_EXTERNAL_LIBS=0
+      - -DENABLE_MPEG=0
+      - -DBUILD_REGTEST=0
+      - -DBUILD_TESTING=0
+    sources:
+      - type: archive
+        url: https://github.com/libsndfile/libsndfile/archive/58c05b87162264200b1aa7790be260fd74c9deee.zip
+        sha256: 3337d28c24f2aaa74b619c9c335db79fd4891d9dda5fa29e70b1c771c46ee342
+
+  # More efficient timestretch with rubberband
+  - name: fftw
```

Comment:
the `double` variant is in the runtieme. The `float` version please use the shared-modules.

Reply:
hmm same, it wasn't picked up by the build system, investigating..